### ISSUE TITLE
[BUG] Using windows, linux module seems to be the culprit?

### ISSUE THREAD

**Issue by zwartkop2** [Batcastle/Monika-TTS#2]
[traceback.txt](https://github.com/Batcastle/Monika-TTS/files/7794756/traceback.txt)

![2021-12-30](https://user-images.githubusercontent.com/79435778/147791540-eab8b118-fdae-45ff-bab5-04f0a7c24e05.png)

Got this right upon booting the game after installing the submod for the first time

**Comment by Batcastle** [comment]
Did you download the Monika-TTS.zip file form the releases page or go to Code > Download Zip?

**Comment by Batcastle** [comment]
I see what is happening with this bug. It looks like, at least in part, Windows using backslashes has yet again caused a programmer pain.

**Comment by Batcastle** [comment]
Github auto-closed this. Needs testing first before we can officially close this issue.

**Comment by Batcastle** [comment]
Testing in a Windows VM indicates this fix is working correctly. If you continue to have issues, please open a new issue.
